### PR TITLE
Stylesheets section

### PR DESCRIPTION
**Pull request by espinr**
A first proposal of the **styling section** (CSS), based on the outline proposed in PR https://github.com/w3c/miniapp-components/pull/6.

***
<a href="https://pr-preview.s3.amazonaws.com/espinr/miniapp-components/pull/9.html" title="Last updated on Jan 26, 2023, 9:42 AM UTC (358f3ea)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-components/9/39b4830...espinr:358f3ea.html" title="Last updated on Jan 26, 2023, 9:42 AM UTC (358f3ea)">Diff</a>

### CODE REVIEW

**Comment by espinr** [comment]
https://pr-preview.s3.amazonaws.com/espinr/miniapp-components/pull/9.html#sec-stylesheets

File: index.html [hunk 1]
```diff
@@ -1,1642 +1,256 @@
...
+  <section id="sec-stylesheets">
+    <h2>Styling</h2>
+    <p>
+      MiniApp user agents support CSS profiles based on the standard specifications defined by the CSS Working Group Snapshot [[CSS-SNAPSHOT]]. Style sheets of MiniApp components follow the CSS recommendations for syntax, selectors, rules, media queries, fonts, and other elements described in [[CSS-SNAPSHOT]] to specify the presentation of the components.
+    </p> 
+    <p>
+      MiniApp user agents are not expected to implement full support of CSS. Instead, they MAY define specific CSS profiles with subsets of CSS properties, selectors, colors, rules, and other CSS specifications, aiming to simplify the development process and meet all their concrete requirements. 
+    </p>  
+    <p>
+      Every MiniApp component has a style sheet (with the `.css` extension) associated by default. Developers MAY include additional style sheets to reuse common styles among the components (e.g., styles for buttons or dialogs). The shared style sheets SHOULD be hosted in the `/Common` directory in the package.
```
Comment (xfq) [inline index.html]: Should it be `/common` (lowercase)?

File: index.html [hunk 2]
```diff
@@ -1,1642 +1,256 @@
...
+    </p>
+    <p>
+      Developers may use the <a href="https://www.w3.org/TR/css-cascade-4/#at-ruledef-import" data-link-type="dfn">@import</a> rule to import style rules from other style sheets. If an <a href="https://www.w3.org/TR/css-cascade-4/#at-ruledef-import" data-link-type="dfn">@import</a> rule refers to a valid style sheet hosted locally in the package, user agents treat the imported style sheet as if its contents were written in place of the @import rule defined in [[CSS-CASCADE-4]].
+    </p>
+    <pre class="example" title="Import of style sheet from another style sheet"><code>/** scoreboard.css **/
+    @import  "../Common/buttons.css";
+
+    .checkout  {
+      color: yellow; 
+    }</code></pre>
```
Comment (xfq) [inline index.html]:
```suggestion
@import  "../Common/buttons.css";

.checkout  {
  color: yellow; 
}</code></pre>

```

File: index.html [hunk 3]
```diff
@@ -1,1642 +1,256 @@
...
+
+    <p data-cite="HTML">
+      Developers MAY also use the `script` element, equivalent to the [^script^] HTML element, to include CSS declarations in the primary HTML resource of the component.
```
Comment (xfq) [inline index.html]:
```suggestion
      Developers MAY also use the `style` element, equivalent to the [^script^] HTML element, to include CSS declarations in the primary HTML resource of the component.  

```

File: index.html (hunk 1 above)
Comment (espinr) [reply index.html]: Done: 8194f26f034e30ac67fa1c250c8e61b0af1b65f5
Thanks, @xfq

**Comment by espinr** [comment]
I've merged all the branches (one per individual section) into one to have a full picture of the document: https://raw.githack.com/espinr/miniapp-components/all-new-sections/index.html#sec-stylesheets

**Review by Sharonzd — APPROVED** [review]

**Pull request merged**